mongo-store.js

```diff
@@ -2,27 +2,18 @@
 'use strict'
 
 const Mongo = require('mongodb')
-const Dot = require('mongo-dot-notation')
 const MongoClient = Mongo.MongoClient
 
-const name = 'mongo-store'
+const { intern } = require('./lib/intern')
 
-const {
-  ensure_id,
-  makeid,
-  idstr,
-  fixquery,
-  metaquery,
-  makeent,
-  should_merge,
-} = require('./lib/common')
+const name = 'mongo-store'
 
 /*
 native$ = object => use object as query, no meta settings
 native$ = array => use first elem as query, second elem as meta settings
 */
 
-module.exports = function (opts) {
+const mongoStore = function (opts) {
```

Comment:
use:
`const mongo_store = function mongo_store(options) {`

this updates the names to be consistent with standard usage:
- the name of the plugin function is significant - Seneca uses if for the plugin name in logs (until overridden by exports)
- plugin options should always be called `options` for consistency
- the const variable should have the same name as the function